app/scripts/controllers/HomeCtrl.js

```diff
@@ -20,7 +20,8 @@
 
     home.sendMessage = function() {
         home.newMessage.roomId = home.currentRoom.$id;
-        home.newMessage.username = home.currentUser;
+        home.newMessage.username = $cookies.get('blocChatCurrentUser');
+        home.newMessage.sentAt = firebase.database.ServerValue.TIMESTAMP;
         Message.send(home.newMessage);
```

Comment:
It could be good here to clear out the message box, otherwise the user could just keep submitting the same message over and over.